### ISSUE TITLE
private key CSV export results in broken newlines on windows

### ISSUE THREAD

**Issue by wchristian** [spesmilo/electrum#2688]
I ran it on windows and it generated \r\r\n newlines instead of just \r\n.

**Comment by LeaTex** [comment]
Right, I test it in version 2.8.3 and 2.9.2 and both are the same.

![electrumcsv292](https://user-images.githubusercontent.com/6115573/28997183-bc076d82-79e4-11e7-87dc-542867d1ba60.png)
